### PR TITLE
GitHub: Issue and PR templates

### PR DESCRIPTION
**Pull request by markmandel**
Three templates:

1. Bug report template
2. Feature request template
3. Pull request template

Lifted these from Agones, and tweaked to adjust for this project.

Feedback definitely welcome!

### CODE REVIEW

**Review by iffyio — APPROVED** [review]

**Comment by markmandel** [comment]
Thanks for the approval!

Just making a note to not merge this until after we do our 0.2.0 release.

**Pull request merged**
